app/lib/vsphere_os_identifiers/data.yaml

```diff
@@ -436,103 +436,163 @@ vmwarePhoton64Guest:
 
 win2000AdvServGuest:
   :description: Microsoft Windows 2000 Advanced Server
+  :name: windows
 win2000ProGuest:
   :description: Microsoft Windows 2000 Professional
+  :name: windows
 win2000ServGuest:
   :description: Microsoft Windows 2000 Server
+  :name: windows
 win31Guest:
   :description: Microsoft Windows 3.1
+  :name: windows
 win95Guest:
   :description: Microsoft Windows 95
+  :name: windows
 win98Guest:
   :description: Microsoft Windows 98
+  :name: windows
 winLonghorn64Guest:
   :description: Microsoft Windows Longhorn (64-bit)
   :architecture: x86_64
   :since: 2.5
+  :name: windows
 winLonghornGuest:
   :description: Microsoft Windows Longhorn (32-bit)
   :architecture: i386
   :since: 2.5
+  :name: windows
 winMeGuest:
   :description: Microsoft Windows Millenium Edition
+  :name: windows
 winNTGuest:
   :description: Microsoft Windows NT
+  :name: windows
 winNetBusinessGuest:
   :description: Microsoft Windows Small Business Server 2003
+  :name: windows
 winNetDatacenter64Guest:
   :description: Microsoft Windows Server 2003 Datacenter Edition (64-bit)
-  :architecture: x86_64
+  :architecture: x64
```

Comment:
This should be the architecture as described in facts, can this stay as `x86_64`?

Reply:
On our hosts, Windows reports itself as being the x64 architecture in facts, though hardware seems to still report x86_64. 

```
> host.facts['os::name']
=> "windows"
> host.facts['os::architecture']
=> "x64"
> host.facts['os::hardware']
=> "x86_64"
> host.architecture.name
=> "x64"
```

Reply:
Ok, thanks!
I'm wondering if this is actually a bug in Foreman's fact parser.
But let's leave this for now at `x64`.

Can you also post host.facts['os::release::full']? That might become valuable if I decide to improve unit tests for this :-)

Reply:
Windows has a very simple os fact hash actually;
```
> host.facts.select { |k, v| k.start_with?('os::') && !v.nil? }
=> {"os::architecture"=>"x64", "os::family"=>"windows", "os::hardware"=>"x86_64", "os::name"=>"windows", "os::release::full"=>"2016", "os::release::major"=>"2016", "os::windows::system32"=>"C:\\Windows\\system32"}
```

Haven't found any hosts where os::release::major and os::release::full aren't the same value, so not sure myself which is the better choice, decided on full since that seemed more apt.